wrapper/src/main/java/software/amazon/jdbc/util/PropertyUtils.java

```diff
@@ -116,4 +123,21 @@ public static void setPropertyOnTarget(
     }
     return copy;
   }
+
+  public static String logProperties(final Properties props, final String caption) {
+    StringBuilder sb = new StringBuilder();
+    for (Object key : props.keySet()) {
+      if (sb.length() > 0) {
+        sb.append("\n");
+      }
+      sb.append("[").append(key.toString()).append("] ").append(props.get(key).toString());
```

Comment:
NIT: wouldn't it be nicer to use string interpolation?

Reply:
> NIT: wouldn't it be nicer to use string interpolation?

Using String.format will have an impact of the performance.